Components/Modal/modal.css

```diff
@@ -0,0 +1,44 @@
+.modal-wrapper {
+  position: fixed;
+  display: none;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgb(0, 0, 0, 0.7);
+  z-index: 10000;
+  color: var(--white);
+}
+
+.btn.modal-btn {
+  color: var(--white);
+}
+
+.modal-container {
+  position: relative;
+  width: 50%;
+  height: fit-content;
+  background-color: var(--primary-color);
+  padding: 15px;
+}
+
+.modal-btn-wrapper {
+  display: flex;
+  justify-content: flex-end;
+  gap: 20px;
```

Comment:
relative values can be used.

Reply:
didnt want it to scale but will change it!